docs/interactions/Application_Commands.md

```diff
@@ -317,7 +317,7 @@ To allow for fine-tuned access to commands, application command permissions are
 Similar to how threads [inherit user and role permissions from the parent channel](#DOCS_TOPICS_THREADS/permissions), any command permissions for a channel will apply to the threads it contains.
 
 > info
-> If you don't have permission to use a command, they'll show up in the command picker as disabled and unusable. They will **not** currently be hidden (though they will in the future).
+> If you don't have permission to use a command, they will not show up in the command picker. Members with the Administrator permission can use all commands.
```

Comment:
Maybe we could just add a note about Android? Either including it in this info box or below it in something like:

> warn
> On Android clients, commands that users don't have access to will appear in the command picker, but will be disabled and unusable.

I don't think it needs to be too specific—this should be fixed in the next month in which I'll remove it.

Reply:
It still wouldn't be accurate as to the behaviour of commands v2. As far as I can tell, on commands v2, the client itself caches the requests it makes for up to 1 hour, which means that it's possible for a command to still take an hour to roll out.

Reply:
> Maybe we could just add a note about Android?

added